testsupport/wait/host.go

```diff
@@ -243,6 +243,27 @@ func (a *HostAwaitility) UpdateUserSignup(userSignupName string, modifyUserSignu
 	return userSignup, err
 }
 
+// UpdateUserSignup tries to update the Spec of the given UserSignup
+// If it fails with an error (for example if the object has been modified) then it retrieves the latest version and tries again
+// Returns the updated UserSignup
```

Comment:
copy-paste driven development :smile: 
```suggestion
// UpdateSpace tries to update the Spec of the given Space
// If it fails with an error (for example if the object has been modified) then it retrieves the latest version and tries again
// Returns the updated Space
```

Reply:
ah damn'it! I thought I changed all occurrences of `UserSignup` 🤦‍♂️